.travis.yml

```diff
@@ -1,6 +1,12 @@
 language: node_js
 node_js:
-  - "0.11"
+  - "node" # latest stable Node.js release
+  - "7"
+  - "6"
+  - "5"
+  - "4"
+  - "0.12"
```

Comment:
I think you can leave out 5 and 7 here. The officially supported node versions are 4 (maintenance), 6 (LTS) and 8 (stable).